src/main/java/tests/ShopServiceTest.java

```diff
@@ -0,0 +1,110 @@
+package tests;
+
+import core.basesyntax.*;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ShopServiceTest {
+
+    private ShopService shopService;
+    private List<FruitTransaction> transactions;
+
+    @BeforeEach
```

Comment:
The `setUp` method should not begin with an empty line according to the checklist. Please remove the redundant empty line at the start of the method implementation.